projects/core/src/occ/services/occ-endpoints.service.ts

```diff
@@ -49,6 +49,30 @@ export class OccEndpointsService {
     return this.config.backend.occ.baseUrl + endpoint;
   }
 
+  /**
+   * Returns an endpoint starting from the OCC prefix (no baseSite)
```

Comment:
Would be good to be a bit more descriptive here. Like:

> Returns an endpoint starting from the OCC prefix, i.e. 
> 
> `/occ/v2/{endpoint}`
> 
> Most OCC endpoints are related to a baseSite context and are therefor prefixed
> with the baseSite. The `/basesites` endpoint does not relate to a specific baseSite
> as it will load all baseSites. 
> 
> @param endpoint Endpoint suffix